modules/oneKeyRtdProvider.md

```diff
@@ -0,0 +1,126 @@
+## OneKey Real-time Data Submodule
+
+The OneKey real-time data module in Prebid has been built so that publishers
+can quickly and easily setup the OneKey Addressability Framework.
+This module is used along with the oneKeyIdSystem to pass OneKey data to your partners.
+Both modules are required. This module will pass transmission requests to your partners
+while the oneKeyIdSystem will pass the oneKeyData.
+
+Background information:
+- [prebid/addressability-framework](https://github.com/prebid/addressability-framework)
+- [prebid/paf-mvp-implementation](https://github.com/prebid/paf-mvp-implementation)
```

Comment:
I don't believe these are stable locations, please link to the new locations or delete temporarily until the location is determined

Reply:
Hi @patmmccann,

For now, there is no other location. As far as I know, nothing is planned for changing them. If later, those locations change, we will update this document accordingly.

Thanks

Reply:
If you don't move the repo, i don't plan to merge, sorry but this is the result of the prebid board vote.

Reply:
Ok, my team wasn't aware of this aspect of the vote. Following your feedback and some clarifications, @BasileLeparmentier has just done the necessary and moved the repositories to another workspace.

I have just updated the locations in this PR accordingly.

Thanks